pkg/recovery/apiserver.go

```diff
@@ -189,7 +189,7 @@ func (s *Apiserver) Create() error {
 	}
 
 	s.recoveryResourcesDir = filepath.Join(s.StaticPodResourcesDir, "recovery-kube-apiserver-pod")
-	err = os.Mkdir(s.recoveryResourcesDir, 755)
+	err = os.Mkdir(s.recoveryResourcesDir, 0750)
```

Comment:
wow, this was an actual bug :+1:

Reply:
@sttts why? the folder contains both configmaps and secrets so it should be public. The actual secret files should be private an I hope they are.

Reply:
ok, so the octal vs. decimal is a bug, not the 5 at the end

Reply:
@sttts @tnozicka while they "should" be private, why is it harmful to still reduce the permissions given to this directory?